docs/faq/sysadmin.rst

```diff
@@ -131,7 +131,9 @@ network at a system level, such that when users invoke ``mpirun`` or
 ``mpiexec`` to launch their jobs, they will automatically only be using
 the network meant for MPI jobs.
 
-:doc:`See the run-time tuning FAQ category </faq/tuning>` for information on how to set global MCA parameters.
+:ref:`See the run-time tuning FAQ category
```

Comment:
FWIW: I did absolute minimal edits on the FAQ pages just so that Sphinx wouldn't warn/error.  The intent is that the FAQ pages will all ultimately go away (i.e., be folded into the rest of the docs) ...which has been happening very slowly.  But your points are valid -- I'll make these changes, because at least for now, these pages are available publicly/google-able.